Feat:  Add Message and Link for ToastBar #DS-1213 (#1426)
lmc-eu/spirit-design-system: packages/web/src/scss/components/Toast/README.md

```html
<div class="ToastBar ToastBar--inverted">
  <div class="ToastBar__box">
    <div class="ToastBar__container">
      <svg width="20" height="20" aria-hidden="true">
        <use xlink:href="/icons/svg/sprite.svg#info" />
      </svg>
      <div class="ToastBar__content">

>         <div class="text-truncate-multiline" data-spirit-populate-field="message">Message with icon</div>

Reviewer: There are no other "populate-fields" here, please remove this attribute from this example.

      </div>
    </div>
  </div>
</div>
```

### ToastBar Components

> The content of `ToastBar` can be assembled from the following components:

Reviewer: ```suggestion
The content of `ToastBar` can be assembled from the following subcomponents:
```

#### ToastBarMessage

> `ToastBarMessage` is a component designates for main message in `ToastBar`.

Reviewer: ```suggestion
`ToastBarMessage` is a subcomponent designated for the main message in `ToastBar`.
```

Usage example:

```html
<div class="ToastBar ToastBar--inverted">
  <div class="ToastBar__box">
    <div class="ToastBar__container">
      <div class="ToastBar__content">

>         <div class="text-truncate-multiline" data-spirit-populate-field="message">Message with action</div>

Reviewer: again, please without the populate attribute. Leave it only in the section where we describe the templating.

      </div>
    </div>
  </div>
</div>
```

> #### API

Reviewer: I think we don't do API tables in the web package. As there is no API.

| Name       | Type     | Default | Required | Description                    |
| ---------- | -------- | ------- | -------- | ------------------------------ |
| `children` | `string` | —       | ✓        | Content of the ToastBarMessage |

#### ToastBarLink

> `ToastBarLink` is a component designated to create an action link within `ToastBar`.

Reviewer: ```suggestion
`ToastBarLink` is a subcomponent designated to create an action link within `ToastBar`.
```

Reviewer: I am not sure if we really do this everywhere. If not, please ignore my suggestion

Usage example:

```html
<div class="ToastBar ToastBar--inverted">
  <div class="ToastBar__box">
    <div class="ToastBar__container">
      <div class="ToastBar__content">
        <div class="text-truncate-multiline" data-spirit-populate-field="message">Message with action</div>

>         <a href="#" class="link-inverted link-underlined ToastBar__link">Action</a>

Reviewer: maybe start with the toastbar__link class? :)

```html
<div id="my-dismissible-toast" class="ToastBar ToastBar--inverted ToastBar--dismissible">
  <div class="ToastBar__box">

>     <div class="ToastBar__container">

Reviewer: you are missing the content div here
